### DISCUSSION TITLE
github copilot: the github server is unable to work with this version of the plugin

### DISCUSSION THREAD

**Discussion by gezmi** [community/community#15671] [Copilot Conversations]
Hi, 

Recently my PyCharm was updated to 2022.1 and a dew days after, I started getting this error:
The GitHub server is unable to work with this version of the plugin. Completions will be turned off until the next restart. Please update to the latest version.

I checked for update and restarted the IDE but it was not resolved, I am unable to use CoPilot. Could you please help me fix this?

Thank you!

**Comment by davecheney** [comment]
I see no telemetry for you for the last 24 hours. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by henriqevargas** [reply]
I have the same problem, did you manage to solve it?

**Reply by LRuiRui517** [reply]
I have the same problem too

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by djcraft** [reply]
DNS resolution does not want to be done behind the corporate proxy to github services for plugin version 1.1.24.1610.

A timeout is generated on the address 140.82.121.4:443

But it works perfectly in version 1.1.20.1417 but the service was closed this morning for this version of the plugin.

**Reply by ruanrz** [reply]
I have the same problem too.it works perfectly only in version 1.1.20.1417 on my Pycharm ,but the service was closed  suddenly/

**Reply by Thepas** [reply]
Actually version 1.1.23.1570 seems to work for me;

Edit: same problem with 1.1.23.1570

**Reply by ruanrz** [reply]
now, version 1.1.25.1678 works for me only if I agree with telemetry thing
